### PR TITLE
[bridge] Update abi

### PR DESCRIPTION
**Pull request by longbowlu**
## Description 

update bridge abi for the recent changes in solidity code 

## Test plan 

How did you test the new or updated feature?

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:
- [ ] REST API:

### CODE REVIEW

File: crates/sui-bridge/abi/bridge_vault.json [hunk 1]
```diff
@@ -14,6 +14,33 @@
...
+      }
+    ],
+    "name": "AddressInsufficientBalance",
+    "type": "error"
+  },
+  {
+    "inputs": [],
+    "name": "FailedInnerCall",
+    "type": "error"
+  },
```
Comment (longbowlu) [inline crates/sui-bridge/abi/bridge_vault.json:43]: @Bridgerz  is this expected?

File: crates/sui-bridge/abi/bridge_vault.json [hunk 2]
```diff
@@ -55,6 +82,22 @@
...
+    "inputs": [
+      {
+        "internalType": "address",
+        "name": "token",
+        "type": "address"
+      }
+    ],
+    "name": "SafeERC20FailedOperation",
+    "type": "error"
+  },
```
Comment (longbowlu) [inline crates/sui-bridge/abi/bridge_vault.json:100]: and this @Bridgerz

File: crates/sui-bridge/abi/sui_bridge.json [hunk 3]
```diff
@@ -10,6 +10,42 @@
...
+    "inputs": [
+      {
+        "internalType": "address",
+        "name": "account",
+        "type": "address"
+      }
+    ],
+    "name": "AddressInsufficientBalance",
+    "type": "error"
+  },
```
Comment (longbowlu) [inline crates/sui-bridge/abi/sui_bridge.json:23]: and this @Bridgerz

File: crates/sui-bridge/abi/sui_bridge.json [hunk 4]
```diff
@@ -82,6 +137,17 @@
...
+    "inputs": [
+      {
+        "internalType": "address",
+        "name": "token",
+        "type": "address"
+      }
+    ],
+    "name": "SafeERC20FailedOperation",
+    "type": "error"
+  },
```
Comment (longbowlu) [inline crates/sui-bridge/abi/sui_bridge.json:150]: this @Bridgerz

File: crates/sui-bridge/abi/sui_bridge.json [hunk 5]
```diff
@@ -355,11 +421,6 @@
         "internalType": "address",
         "name": "_limiter",
         "type": "address"
-      },
-      {
-        "internalType": "address",
-        "name": "_wETH",
-        "type": "address"
```
Comment (longbowlu) [inline crates/sui-bridge/abi/sui_bridge.json:362]: this @Bridgerz

File: crates/sui-bridge/abi/sui_bridge.json [hunk 6]
```diff
@@ -564,18 +625,5 @@
...
-    "name": "wETH",
-    "outputs": [
-      {
-        "internalType": "contract IWETH9",
-        "name": "",
-        "type": "address"
-      }
-    ],
-    "stateMutability": "view",
-    "type": "function"
```
Comment (longbowlu) [inline crates/sui-bridge/abi/sui_bridge.json:579]: @Bridgerz

File: crates/sui-bridge/abi/sui_bridge.json (hunk 5 above)
Comment (Bridgerz) [reply crates/sui-bridge/abi/sui_bridge.json:362]: `SuiBridge` no longer has a reference to weth as the vault handles all wrapping and unwrapping. must be a stale reference to an old version.

File: crates/sui-bridge/abi/sui_bridge.json (hunk 6 above)
Comment (Bridgerz) [reply crates/sui-bridge/abi/sui_bridge.json:579]: dito

**Review by Bridgerz — APPROVED** [review]

**Pull request merged**